### ISSUE TITLE
@coverage annotation for function outside class

### ISSUE THREAD

**Issue by darthf1** [phpstan/phpstan-phpunit#143]
Hi!

With an update to `1.2.1` I get a new error reported regarding my `@coverage` annotation. The coverage is reported correctly for this function, however PHPStan tells me the annotation is incorrect.

![image](https://user-images.githubusercontent.com/17253332/198524990-59dec70e-972d-4d52-a7e5-09aeb59b1c7c.png)

Functions file:
```php
<?php

declare(strict_types=1);

namespace Domain\Shared\Translation;

/**
 * @param array<mixed> $parameters
 */
function t(string $message, array $parameters = [], string $domain = null): TranslatableMessage {}
```

Test class:

```php
<?php

declare(strict_types=1);

namespace Tests\Unit\Domain\Shared\Translation;

use Tests\Unit\UnitTestCase;

use function Domain\Shared\Translation\t;

/**
 * @covers \Domain\Shared\Translation\t
 */
class FunctionsTest extends UnitTestCase
{}
```

`composer.json`:
```
{
    "autoload": {
        "files": [
            "src/Domain/Shared/Translation/functions.php"
        ],
        "psr-4": {
            "Application\\": "src/Application",
            "Domain\\": "src/Domain",
            "Infrastructure\\": "src/Infrastructure",
            "UserInterface\\": "src/UserInterface"
        }
    },
}
